.github/workflows/codeguru-reviewer.yml

```diff
@@ -0,0 +1,40 @@
+name: Run CodeGuru Reviewer
+
+on: [ push, pull_request, workflow_dispatch ]
```

Comment:
Here we use `pull_request` instead of `pull_request_target`. Won't his cause PR started from fork not able to run this workflow as the workflow being executed is actually from their fork?